lib/src/data/chat/chat_message.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:misskey_dart/misskey_dart.dart';
+import 'package:misskey_dart/src/converters/date_time_converter.dart';
+
+part 'chat_message.freezed.dart';
+part 'chat_message.g.dart';
+
+@freezed
+class ChatMessage with _$ChatMessage {
+  const factory ChatMessage({
+    required String id,
+    @DateTimeConverter() required DateTime createdAt,
+    String? text,
+    required String fromUserId,
+    User? fromUser,
+    String? toUserId,
+    User? toUser,
+    String? toRoomId,
+    ChatRoom? toRoom,
```

Comment:
```suggestion
    UserLite? fromUser,
    String? toUserId,
    UserLite? toUser,
    String? toRoomId,
    ChatRoom? toRoom,
    String? fileId,
    DriveFile? file,
    bool? isRead,
```

ref: https://github.com/misskey-dev/misskey/blob/2025.4.0-beta.0/packages/backend/src/models/json-schema/chat-message.ts#L22-L61